backend/app/logins.py

```diff
@@ -849,7 +850,10 @@ def continue_oauth_flow(
         # in then create a user
         user = login.user
         if user is None:
-            user = models.FlathubUser(display_name=provider_data["name"])
+            user = models.FlathubUser(
+                display_name=provider_data["name"],
+                invite_code=secrets.token_urlsafe(6),
```

Comment:
Wondering, if we could generate a word combination instead? `styled hatchet stoke blooms` for e.g. and I guess it could fail, when we insert here?

Reply:
Hmm, not sure if the UX is much better, especially for i18n? We should at least split the characters into chunks though.

Reply:
Yeah, that sounds like a neat compromise